rust/processor/src/db/common/models/fungible_asset_models/v2_fungible_asset_balances.rs

```diff
@@ -165,6 +165,12 @@ impl FungibleAssetBalance {
                 let asset_type = inner.metadata.get_reference_address();
                 let is_primary = Self::is_primary(&owner_address, &asset_type, &storage_id);
 
+                // If store has the untransferable field then it must be frozen
+                let is_frozen = if object_data.untransferable.as_ref().is_some() {
+                    true
+                } else {
+                    inner.frozen
```

Comment:
in the move object, what's the difference between frozen and untransferable?

Reply:
this needs to be removed.